.gitignore

```diff
@@ -35,6 +35,7 @@ tags
 
 # Editors
 .vscode
+CMakePresets.json
```

Comment:
Why we need this?

Reply:
This is the vscode config file I use for cmake

Reply:
removed in latest commit